static/styles/settings.css

```diff
@@ -38,6 +38,35 @@ h3,
     }
 }
 
+/* dropdown design */
+.dropdown-wrap {
+    position: relative;
+    width: fit-content;
+
+    select {
+        appearance: none;
+        margin: 0;
+    }
+
+    &::before,
+    &::after {
```

Comment:
This is great, but we would still want to use `fa fa-chevron-down` icon here so that we are using the same icon to represent it across the app.